### ISSUE TITLE
Impossible to Read Monster Names

### ISSUE THREAD

**Issue by estebandellasilva** [CleverRaven/Cataclysm-DDA#57826]
### Describe the bug

Found a Zombie or Monster i cannot read the name of
Its probably the regenerating zombie Line

### Steps to reproduce

Find regenerating zombie medic that cannot be named (according to the discord)
Try to read the name

### Expected behavior

Color Combo to be readable

### Screenshots

![image](https://user-images.githubusercontent.com/89189872/169638986-584c074a-b090-4f9f-84eb-8a4a23c041ed.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19042.1645 (20H2)
- Game Version: 83d243d [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    No Rail Stations [No_Rail_Stations],
    Stats Through Kills [stats_through_kills],
    SpeedyDex [speedydex]
]

**Comment by GuardianDll** [comment]
would be good to change `pink_red` color of `mon_zombie_regenerating` to `i_pink_red`, but in some reason it doesnt work

**Comment by Nindows** [comment]
I changed the color to brown_cyan. This change seems to work. Color combination is not used by any other monster either. I also tried i_pink_red. It does not work like GuardianDll said. It seems that colors and especially color combinations have to be defined in color.cpp first
![test](https://user-images.githubusercontent.com/100204032/171067212-167601f2-d3ea-4522-8b02-0690cd5d2e2b.png)
.

**Comment by Nindows** [comment]
There are also other instances of hard to read monster names. I think I included everyone here. Is it possible to change text without changing the monster icon in curse? Because if you change the color json you also change the icon of the monster in game.
![hard_to_read](https://user-images.githubusercontent.com/100204032/171387385-9dfad46e-4db5-477e-814b-b8e60b4dd5fd.png)
